docs/tizen-distribution/how-distribute/sales.md

```diff
@@ -0,0 +1,35 @@
+# Sales
+
+## 1) Country & Price
+
+- You can change the prices of your application without any validation even after you start selling the application. However, you can change countries for sales only when an update occurs.
+
+![img](media/distribution_sales01.jpg)
+
+1. Select the ’Free’ if you want to provide your application free of charge.
+2. If you would like to sell paid applications, deselect the checkbox and enter a standard price in US dollars. When you click [Auto convert local currency], the price for sales will be automatically calculated and entered based on the set exchange rate for each country. If a country requires a different policy for price, you can change the sale price manually. (*You can change the prices of your application without any validation even after you start selling the application. However, you can change countries for sales only when an update occurs.)
+3. You can select whether or not to sell applications when more countries are added for this service and sales. Select “Select All Countries”, or select/deselect a checkbox for the country of your choice.
+4. If the Phone Bill price is different for each service provider in a particular country, the Phone Bill price is offered in Tiers. If you click “Carrier Price”, you can view the Phone Bill price for each service provider. If the entered price is not supported in Phone Bill, the nearest price is suggested at the bottom. If you do not enter a suggested value from Tier or the recommended text, Phone Bill will not be available as a payment method. (On the screen for price selection, the highest price in each tier is offered as the display price.)
+5. In countries where the Tier-method of Phone Bills is supported, when an auto-converted price is less than the standard price of USD 5, the price charged will be the closest Tier price, based on the auto-converted figure (+/- 5% of the auto-converted amount is allowed).
+6. When you enter a price for applications, an open-lock icon will be shown. When you click the lock-shaped icon to change it to a locked icon, the set price cannot be changed. Also, you cannot change the price for the country when you click the [Auto convert local currency] button. If you need to change the price, simply click on the lock-shaped icon.
+
+[![img](media/sales2-330x99.png)](media/sales2.png)
```

Comment:
The image has to line up with the 6th item of ordered list.
Please indent the image.